src/devices/SensorHub/SensorHub.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.SensorHub
+{
+    /// <summary>
+    /// SensorHub
+    /// </summary>
+    public class SensorHub
+    {
+        /// <summary>
+        /// Default I2C bus id
+        /// </summary>
+        public const int DefaultI2cBusId = 1;
```

Comment:
Is the device a Hat for a specific board? If not then I'd recommend to remove this

Reply:
done